python/lsst/pipe/base/argumentParser.py

```diff
@@ -1039,7 +1038,7 @@ def write(self, showStr):
             matHistory = re.search(r"^(?:config.)?(.+)?", showArgs)
             globPattern = matHistory.group(1)
             if not globPattern:
-                print("Please provide a value with --show history (e.g. history=*.doXXX)", file=sys.stderr)
+                sys.stderr.write("Please provide a value with --show history (e.g. history=*.doXXX)\n")
```

Comment:
I don't like this at all. What's the justification given that `file=sys.stderr` is a good option already.

Reply:
The justification is that the existing code is a mix of the two styles (even within the same module). I feel write is a bit cleaner, though it does risk forgetting the final \n. In any case I think we should pick one and stick to it. I'm willing to switch to print if you feel strongly about it. I find the mix confusing.

Reply:
Since you prefer print and I don't have strong feelings either way I updated the code to use print everywhere. It all appeared to be in the one module.